### PR TITLE
Bluetooth: GATT: Add BTP Discover All Primary Services command handler

### LINKED ISSUE #17861: Tester application lacks BTP Discover All Primary Services handler
**Describe the bug**
The GATT Discover All Primary Services was introduced in zephyrproject-rtos/zephyr#9986.
The tester application still lacks support for it.

**Impact**
Cannot automate GATT/CL/GAD/BV-01-C test case.

### PR DESCRIPTION
**Pull request by MariuszSkamra**
This will handle BTP Discover All Primary Services command.

Related TC: GATT/CL/GAD/BV-01-C
Fixes: #17861
Signed-off-by: Mariusz Skamra <mariusz.skamra@codecoup.pl>

### CODE REVIEW

**Comment by ioannisg** [comment]
Does this PR need to be back-ported?

**Comment by joerchan** [comment]
Backpoirt is required, added backport label.

File: tests/bluetooth/tester/src/gatt.c [hunk 1]
```diff
@@ -919,17 +920,17 @@ static void discover_destroy(struct bt_gatt_discover_params *params)
 	gatt_buf_clear();
 }
 
-static u8_t disc_prim_uuid_cb(struct bt_conn *conn,
+static u8_t disc_prim_cb(struct bt_conn *conn,
 				 const struct bt_gatt_attr *attr,
```
Comment (joerchan) [inline tests/bluetooth/tester/src/gatt.c]: realign

File: tests/bluetooth/tester/src/gatt.c [hunk 2]
```diff
@@ -919,17 +920,17 @@ static void discover_destroy(struct bt_gatt_discover_params *params)
...
 				 struct bt_gatt_discover_params *params)
```
Comment (joerchan) [inline tests/bluetooth/tester/src/gatt.c]:
```suggestion
			 struct bt_gatt_discover_params *params)
```

**Review by joerchan — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**